Misc/NEWS.d/next/Build/2025-03-15-12-32-56.gh-issue-131278.1nd0mJ.rst

```diff
@@ -0,0 +1,2 @@
+Add optimizing flag ``WITH_COMPUTED_GOTOS`` to support such builds using
+clang-cl on Windows. Patch by Chris Eibl.
```

Comment:
```suggestion
:prog:`clang-cl` on Windows. Patch by Chris Eibl.
```

The `prog` role is a bit unknown but it may help in the future if we change how it's being rendered.